nnf/tseitin.py

```diff
@@ -10,10 +10,11 @@
 from nnf.util import memoize
 
 
-def to_CNF(theory: NNF) -> And[Or[Var]]:
+def to_CNF(theory: NNF, simplify: bool = True) -> And[Or[Var]]:
     """Convert an NNF into CNF using the Tseitin Encoding.
 
     :param theory: Theory to convert.
+    :param simplify: If True, remove clauses that are always true.
```

Comment:
```suggestion
    :param simplify: If True, simplify clauses even if that means eliminating
                     variables.
```
This clarifies why you'd want it to be `False`.